src/main.cpp

```diff
@@ -15,5 +18,11 @@ int main(){
 	std::cout << "checking wrong password: " << std::flush
 			  << BCrypt::validatePassword(wrong_password,hash) << std::endl;
 
+
+    char source[] = "jta{;CD+V8[tpsVhCl0+XJ";
+    char dest[strlen(source)];
+    generateSpecialHash(source, dest);
+
```

Comment:
Is the cout just for debugging?

Reply:
Yes, this is a handcrafted input which results in a special string - I think the intended receiver @Ka0o0 already figured that out :)

Reply:
Nice Idea ;)